Dockerfile.Nextcloud

```diff
@@ -1,8 +1,36 @@
 ARG NEXTCLOUD_VERSION=29.0.4
+ARG INSTALL_XDEBUG=1
 
-FROM nextcloud:${NEXTCLOUD_VERSION}
+FROM nextcloud:29.0.4
 
-RUN apt update && apt install -y less vim telnet iputils-ping
+RUN apt update && apt install -y \
```

Comment:
Most of the changes are for helping us vie coredump files. You have to activate commented Out stuff, build the Container and then Install the /tmp/apache2-bin_* and /tmp/apache2-bin-dbgsym_* packages via apt Install ./package.deb

Then you can actually `gdb /usr/sbin/apache2 coredumpfile`